siriuspy/siriuspy/devices/dvf.py

```diff
@@ -0,0 +1,99 @@
+"""DVF devices."""
+
+import numpy as _np
+
+from .device import DeviceNC as _DeviceNC
+
+
+class DVF(_DeviceNC):
+    """."""
```

Comment:
maybe a docstring here with the meaning of the abbreviation? `Dispositivo de visualização de feixe`

Reply:
I am lazy. I have bertrand russell to back me up. anyway, done.